runtime/list_test.go

```diff
@@ -220,6 +220,40 @@ func TestListAppend(t *testing.T) {
 	}
 }
 
+func TestListExtend(t *testing.T) {
+	fun := newBuiltinFunction("TestListExtend", func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+		app, raised := GetAttr(f, ListType.ToObject(), NewStr("extend"), nil)
+		if raised != nil {
+			return nil, raised
+		}
+		if _, raised := app.Call(f, args, nil); raised != nil {
+			return nil, raised
+		}
+		return args[0], nil
+	}).ToObject()
+	cases := []invokeTestCase{
+		{args: wrapArgs(newTestList().ToObject(), newTestTuple().ToObject()), want: newTestList().ToObject()},
```

Comment:
No need to convert ToObject() inside of wrapArgs, newTestTuple, newTestList, etc. Those methods take interface{} and do any conversions necessary.